src/ci/docker/run.sh

```diff
@@ -37,6 +37,9 @@ if [ -f "$docker_dir/$image/Dockerfile" ]; then
       echo "Downloaded containers:\n$loaded_images"
     fi
 
+    prepare_context="${docker_dir}/${image}/prepare-context.sh"
```

Comment:
This seems a bit overly magical, could the LLVM sources just be downloaded in the container?

Reply:
Ok. @Mark-Simulacrum specifically requested this change. Review c5c09da2c1f89f19554f3bfaff09f48decba25cb if you want that behavior.

Reply:
That was a comment about reusing git commits, which may be nice to use but I don't think it should require magical scripts like this. Everything about docker layers is already cached, so it's best to just go ahead and download separately here as it likely doesn't need the exact same revision of LLVM we use anyway.

Reply:
I also missed that this was in the *setup* for the docker containers so thought it wouldn't be cached.

Reply:
We have reverted the changes to copy existing `llvm/cmake` into the container.
Also, 2 days back, llvm-project to monorepo. Accordingly, we have moved our libunwind changes to [Fortanix's fork of the llvm-project](https://github.com/fortanix/llvm-project/).
In the recent push, we are using the same to build libunwind.a.
To optimise the docker build time, we are downloading source of the specific commit we need from the above port. The implementation to do the same is based on `fetch_submodule` from init_repo.sh)